rules/libyang.mk

```diff
@@ -1,23 +1,31 @@
 # libyang
 
-LIBYANG_VERSION_BASE = 0.16
-LIBYANG_VERSION = $(LIBYANG_VERSION_BASE).105
+LIBYANG_VERSION_BASE = 1.0
+LIBYANG_VERSION = $(LIBYANG_VERSION_BASE).70
 LIBYANG_SUBVERSION = 1
 
 export LIBYANG_VERSION_BASE
 export LIBYANG_VERSION
 export LIBYANG_SUBVERSION
 
-LIBYANG = libyang$(LIBYANG_VERSION_BASE)_$(LIBYANG_VERSION)-$(LIBYANG_SUBVERSION)_$(CONFIGURED_ARCH).deb
+#LIBYANG = libyang$(LIBYANG_VERSION_BASE)_$(LIBYANG_VERSION)-$(LIBYANG_SUBVERSION)_$(CONFIGURED_ARCH).deb
+LIBYANG = libyang_$(LIBYANG_VERSION)_$(CONFIGURED_ARCH).deb
```

Comment:
Any reason we want to change the package name?

Reply:
To match the built libyang package name.   Either change the package name here or change the makefile of the libyang for the debian package name

Reply:
We can keep the libang default package name,  and let the dependent party to use the new name.